eng/build-test.yml

```diff
@@ -0,0 +1,183 @@
+parameters:
+  buildConfig: ''
+  archType: ''
+  osGroup: ''
+  osIdentifier: ''
+  container: ''
+  testGroup: ''
+  readyToRun: false
+  crossrootfsDir: ''
+  # If true, run the corefx tests instead of the coreclr ones
+  corefxTests: false
+  displayNameArgs: ''
+  condition: true
+
+### Test build job
+
+### Each test build job depends on a corresponding build job with the same
+### buildConfig and archType.
+
+jobs:
+- template: xplat-job.yml
+  parameters:
+    buildConfig: ${{ parameters.buildConfig }}
+    archType: ${{ parameters.archType }}
+    osGroup: ${{ parameters.osGroup }}
+    osIdentifier: ${{ parameters.osIdentifier }}
+    helixType: 'build/tests/'
```

Comment:
This seems now unused.

Reply:
Sorry, I don't get this, please elaborate. I believe both jobs need the primary build arch / os and other parameters to properly set up the various variables in xplat-job and elsewhere - what am I missing?

Reply:
The review makes it difficult to focus on what I was I guess. `helixType: 'build/tests/'` seems unused in this job.

Reply:
I agree it's unfortunate there's no more explicit way to express that we're actually "splitting test-job in two", at least I'm not aware of any stock GitHub functionality supporting this. Other than that, thanks for pointing out helixType, I'll fix that.